reddit.js

```diff
@@ -2,4 +2,76 @@ var reddit = require("./data.json")
 
 // Write your solutions below
 
-console.log(reddit["data"]);
\ No newline at end of file
+console.log("Question 1: ***************************")
+
+reddit.data.children.forEach(function (child, idx) {
+	console.log(child.data.title);
+	console.log(idx);
+});
+
+// console.log(reddit["data"]);
+
+console.log("Question 2: ***************************")
+
+var permalink = reddit.data.children.map(function (child) {
+	return "http://reddit.com" + child.data.permalink 
+
+});
+
+console.log(permalink);
+
+console.log("Question 3: ***************************")
+
+var selfText = reddit.data.children.filter(function (selfTextObject) {
```

Comment:
Here, we are iterating over children still, so your parameter can simply be called ```child``` as with the others above.